src/config/config.zig

```diff
@@ -50,8 +62,10 @@ pub const Config = struct {
             .allocator = allocator,
             .rpc_port = 8332,
             .p2p_port = 8333,
-            .testnet = false,
+            .network_id = BitcoinNetworkId.MAINNET,
             .datadir = ".bitcoin",
+            .services = 0,
+            .protocol_version = PROTOCOL_VERSION,
         };
```

Comment:
Better for this to be:

``zig
var config = Config{
            .allocator = allocator
        };
```